### ISSUE TITLE
Strange Graphics when reloading movie data

### ISSUE THREAD

**Issue by mark007** [pix/aeon#56]
Using XBMC 32049, latest aeon65 skin as of 22nd of July.

After renaming some files, poster.jpg, to folder.jpg, I had to reload the data for many movies.

When going to a movie information, and hitting reload. I select not to load data from the internet. The following two "corrupt" screens are shown. This is repeatable about 80% of the time.

When reload is pressed, this is seen for maybe 4 seconds, like as if the gui has frozen. -> http://img189.imageshack.us/i/screenshot000tv.jpg/

Then this screen disappears and the following is seen, showing various posters that appear to be unloaded, very strange looking :P -> http://img844.imageshack.us/i/screenshot001v.jpg/

**Comment by pix** [comment]
I think this might be part of an animation which may be very slow on your computer.

**Comment by mark007** [comment]
Hmm, maybe that is the case, but its a Core i7 920 @ 3.7ghz, and a GTX 295..... I don't the majority of users will have much higher specs to be honest....

I think the first animation is not due to speed because one time it will work perfectly smooth, 50.0fps vsync, the next it sits and freezes, as if its blocked waiting on a process (which I think is xbmc going off to see if data exists online) to complete. I don't think this should be the case.

The second with missing images, I don't think speed should cause any image to display incorrectly. OK maybe it can go slow, but at least the textures should be loaded before any image is drawn.

oops the second image I psoted http://img844.imageshack.us/i/screenshot001v.jpg/ didn't show what I wanted. Right before the screenshot was took, approx 3 of the posters were blank.. This is what I was trying to describe.

So in summary, I don't think performance is the issue here, I think something is blocking the drawing of the image when reloading data, and I'll try to get a screenshot of the second problem where some of the textures for posters seem to get unloaded after reloading a movies information.

**Comment by mark007** [comment]
OK I reloaded the data for this movie, the illusionist, and again saw 2 posters display no textures, but strangely when I take a screenshot, none of them seem to have textures.

This disappears after the movie is completely reloaded, but that one or two seconds looks ugly...... http://img24.imageshack.us/img24/766/screenshot000iy.jpg

Its the center poster and the right most poster that displayed as blank this time around (so the screenshot doesn't 100% reflect what I was seeing). I can reproduce again and again so if I catch a screenshot showing exactly the missing texture effect I'll post it.

Actually in the screenshot that didn't exactly show what I meant here http://img844.imageshack.us/i/screenshot001v.jpg/ you can see when reloading the movies data, the fanart of the movie "Shrek the third" is shown instead of the fanart for "The Illusionist" temporarily..... strange stuff goin on :P Doesn't look right at all..

**Comment by sergiocos** [comment]
Shrek is show as part of your backdrops for movies... xbmc assumes that, until reload finishes, there is no legit backdrop to be showed. So ... not a bug.
i noticed to freezes but my money is on xbmc scraping system, not on the skin... i bet i saw  other skin behave exactly like this, 'night' i think. Let's wait until Dharma is stable enough.

**Comment by pix** [comment]
I thinks thats effectively a problem inherent to XBMC. The problem in kind of the same using Confluence. When you reload the information, XBMC temporary lose fanarts in the middle of an a animations. I don't think there a lot of room for improvement. (Again I'm no expert in the subject)

**Comment by phil65** [comment]
not in our hands, closed
